content/webapp/components/ImageGallery/ImageGallery.tsx

```diff
@@ -139,33 +136,18 @@ const Gallery = styled.div.attrs({
     opacity: 0;
     transition: opacity 400ms ease;
 
-    @media (min-width: ${props => props.theme.sizes.medium}px) {
+    ${props =>
+      props.theme.media('medium')(`
       top: 200px;
 
-      ${props =>
-        props.isStandalone &&
-        `
-        top: 0;
-      `}
+      ${props.isStandalone && `top: 0;`}
 
-      ${props =>
-        props.isActive &&
-        `
-        opacity: 0.1;
-      `}
-    }
+      ${props.isActive && `opacity: 0.1;`}
```

Comment:
Do we need these properties for the medium breakpoint? It looks like they're replicating properties that are applied at any breakpoint on L148 and L150?